spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -284,7 +286,7 @@ public void testAutoCreateTopicsEnabledSucceeds() throws Exception {
 		FixedBackOffPolicy backOffPolicy = new FixedBackOffPolicy();
 		backOffPolicy.setBackOffPeriod(1000);
 		metatadataRetrievalRetryOperations.setBackOffPolicy(backOffPolicy);
-		setMetadataRetryOperations(binder, metatadataRetrievalRetryOperations);
+		//setMetadataRetryOperations(binder, metatadataRetrievalRetryOperations);
```

Comment:
I just checked the binder and provisioner and they both now have a field for this data.

It looks like this code should be moved to a utility class if it's needed in both classes.